Cargo.toml

```diff
@@ -7,3 +7,12 @@ edition = "2018"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+ethcontract = "0.7.2"
+ethabi = "12.0.0"
+serde = { version = "1.0", features = ["derive"] }
+multihash = "0.11.4"
+rustc-hex = "2.1.0"
+anyhow = "1.0.32"
+web3 = "0.13.0"
+serde_json = "1.0.57"
+ethsign = "0.7.3"
```

Comment:
I would use `ethcontract::web3::signing::Signature` instead of adding an additional crate dependency for the Signature data-type.